src/evaluator.ts

```diff
@@ -76,9 +80,19 @@ export class Evaluator {
     bucketBy: string,
     percentage: number,
   ): boolean {
-    const property = this.getAttrValue(target, bucketBy);
+    let property = this.getAttrValue(target, bucketBy);
     if (!property) {
-      return false;
+      const old_bucketBy = bucketBy;
+      bucketBy = 'identifier';
```

Comment:
There's technically no reason you can't do this as args are considered to be `var` (as opposed to `let` or `const`), but it's generally a bad idea to mutate args. Maybe better to extract a `let` variable as you did with `property`